### ISSUE TITLE
alt in img tags are white color in sponsors page

### ISSUE THREAD

**Issue by Rinshin-Jalal** [mastodon/joinmastodon#231]
The alt text has white color same as the background

At first, it is taking some time to load some images
So change the color of the alt text in sponsors page

**Comment by Shrinks99** [comment]
Can't seem to reproduce this in Firefox or Chrome... Removed an image and the alt text appears to come up in black.

<img width="554" alt="Screenshot 2022-11-07 at 12 27 13 AM" src="https://user-images.githubusercontent.com/5672810/200232855-0cf4566d-62ec-4e8e-8f29-86e232aea553.png">

Are you talking about another part of the site?  Could you screenshot the issue?

**Comment by Rinshin-Jalal** [comment]
@Shrinks99 

This works sometimes but when I have low speed 
I got this ( After 6secs it comes )
![image](https://user-images.githubusercontent.com/88965873/200235155-1016e825-d881-4fa1-a6b0-33e34d8df982.png)

**Comment by Shrinks99** [comment]
Just guessing but this seems more like your browser rendering information as it loads it and not a bug with the styling?

**Comment by Rinshin-Jalal** [comment]
ok

**Comment by ChaosExAnima** [comment]
the text is definitely black now. closing.
